hedera-node/hedera-app/src/test/java/com/hedera/node/app/workflows/ingest/IngestWorkflowImplTest.java

```diff
@@ -81,529 +81,533 @@
-//                    .isInstanceOf(RuntimeException.class)
-//                    .hasMessageContaining("checkSolvency exception");
-//            // And the transaction is not submitted to the platform
-//            verify(submissionManager, never()).submit(any(), any());
-//            // And the metrics for counting submitted transactions was not incremented
-//            verify(countSubmitted, never()).increment();
-//        }
-//    }
-//
-//    @Nested
-//    @DisplayName("7. Submit to platform")
-//    class PlatformSubmissionTests {
-//
-//        @Test
-//        @DisplayName("If the platform fails to accept the transaction, the transaction should be rejected")
-//        void testSubmitFails() throws PreCheckException, IOException {
-//            // Given a SubmissionManager that will fail the submit
-//            doThrow(new PreCheckException(PLATFORM_TRANSACTION_NOT_CREATED))
-//                    .when(submissionManager)
-//                    .submit(any(), any());
-//
-//            // When we submit a transaction
-//            workflow.submitTransaction(ctx, requestBuffer, responseBuffer);
-//
-//            // Then the response will indicate the platform rejected the transaction
-//            final TransactionResponse response = parseResponse(responseBuffer);
-//            assertThat(response.nodeTransactionPrecheckCode()).isEqualTo(PLATFORM_TRANSACTION_NOT_CREATED);
-//            // And the cost will be zero
-//            assertThat(response.cost()).isZero();
-//            // And the metrics for counting submitted transactions was not incremented
-//            verify(countSubmitted, never()).increment();
-//        }
-//
-//        @Test
-//        @DisplayName("If some random exception is thrown from submitting to the platform, the exception is bubbled up")
-//        void randomException() throws PreCheckException {
-//            // Given a SubmissionManager that will throw a RuntimeException from submit
-//            doThrow(new RuntimeException("submit exception"))
-//                    .when(submissionManager)
-//                    .submit(any(), any());
-//
-//            // When the transaction is submitted, then the exception is bubbled up
-//            assertThatThrownBy(() -> workflow.submitTransaction(ctx, requestBuffer, responseBuffer))
-//                    .isInstanceOf(RuntimeException.class)
-//                    .hasMessageContaining("submit exception");
-//            // And the metrics for counting submitted transactions was not incremented
-//            verify(countSubmitted, never()).increment();
-//        }
-//    }
-//
-//    private static TransactionResponse parseResponse(@NonNull final DataBuffer responseBuffer) throws IOException {
-//        responseBuffer.flip();
-//        return TransactionResponse.PROTOBUF.parse(responseBuffer);
-//    }
+    /** The workflow to be tested. */
```

Comment:
I'm worried about this one too. At least include the commented out code + the new code and I can sort it out.

Reply:
Reverted this to `04712-use-pbj` contents.